src/runners/cairoRunner.ts

```diff
@@ -43,23 +43,21 @@ export class CairoRunner {
 
   // Run until the given PC is reached.
   runUntilPc(finalPc: Relocatable, verbose: boolean = false): void {
-    while (this.vm.runContext.pc.getOffset() < finalPc.getOffset()) {
+    while (this.vm.pc.getOffset() < finalPc.getOffset()) {
       this.vm.step();
       if (verbose) {
         console.log(
-          `AP: ${this.vm.runContext.ap.getSegmentIndex()}:${this.vm.runContext.ap.getOffset()}`
+          `AP: ${this.vm.ap.getSegmentIndex()}:${this.vm.ap.getOffset()}`
         );
         console.log(
-          `FP: ${this.vm.runContext.fp.getSegmentIndex()}:${this.vm.runContext.fp.getOffset()}`
+          `FP: ${this.vm.fp.getSegmentIndex()}:${this.vm.fp.getOffset()}`
         );
         console.log(
-          `PC: ${this.vm.runContext.pc.getSegmentIndex()}:${this.vm.runContext.pc.getOffset()}`
+          `PC: ${this.vm.pc.getSegmentIndex()}:${this.vm.pc.getOffset()}`
         );
 
-        console.log(
-          `[ap - 1]: ${this.vm.memory.get(this.vm.runContext.ap.sub(1))}`
-        );
-        console.log(`[fp]: ${this.vm.memory.get(this.vm.runContext.fp)}`);
+        console.log(`[ap - 1]: ${this.vm.memory.get(this.vm.ap.sub(1))}`);
```

Comment:
is there a better logger for ts?

Reply:
will investigate, and create a tracer epic